tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/tcf_enclave.h

```diff
@@ -19,6 +19,7 @@
 #include <string>
 
 #include "signup_info_singleton.h"
+#include "signup_info_kme.h"
```

Comment:
It would be better to include `signup_info_singleton.h` and `signup_info_kme.h` directly in their respective interface files (`avalon_enclave_singleton.i`, `avalon_enclave_kme.i`) rather than in `tcf_enclave.h`.

Reply:
Makes sense. i will create 3 swig files and include the avalon_enclave_singleton.i in the setup.py as of now. When we have the infrastructure ready for other workers, we shall include the other swig files in respective workers.